src/components/ha-menu-button.ts

```diff
@@ -130,6 +130,7 @@ class HaMenuButton extends LitElement {
     return css`
       :host {
         position: relative;
+        margin-right: 13px;
```

Comment:
We should stick to material guidelines, that means all padding/margin is a multiple of 4px.

So make this 12px